azure-webapp-maven-plugin/README.md

```diff
@@ -242,7 +212,71 @@ Property | Required | Description
 
 Check out samples at [Web App Samples](../docs/web-app-samples.md) for the configuration settings for different image sources.
 
-### Supported Regions
+
+## Deployment Type 
+
+### Deploy via FTP
+You can deploy your **WAR** file and other artifacts/resources to Web App via FTP. The following example shows all configuration elements.
+
+```xml
+<plugin>
+   <groupId>com.microsoft.azure</groupId>
+   <artifactId>azure-webapp-maven-plugin</artifactId>
+   <configuration>
+      <deploymentType>ftp</deploymentType>
+      <resources>
+         <resource>
+             <directory>${project.basedir}/target</directory>
+             <targetPath>webapps</targetPath>
+             <includes>
+                 <include>*.war</include>
+             </includes>
+             <excludes>
+                 <exclude>*.xml</exclude>
+             </excludes>
+         </resource>
+      </resources>
+      ...
+   </configuration>
+</plugin>
+```
+   
+Detailed explanation of the `<resource>` element is listed in the following table.
+
+Property | Description
+---|---
+`directory` | Specifies where the resources are stored.
+`targetPath` | Specifies the target path where the resources will be deployed to.<br>This is a relative path to the `/site/wwwroot/` folder of FTP server in your Web App.
+`includes` | A list of patterns to include, e.g. `**/*.war`.
+`excludes` | A list of patterns to exclude, e.g. `**/*.xml`.
+
+### WAR Deployment 
+You can deploy your **WAR** file with WAR Deployment. The following example shows the configuration elements.
+
+```xml
+<plugin>
+   <groupId>com.microsoft.azure</groupId>
+   <artifactId>azure-webapp-maven-plugin</artifactId>
+   <configuration>
+      <deploymentType>war</deploymentType>
+      
+      <warFile>custom/absolute/path/deploy.war</warFile>
+
+      <path>/${project.build.finalName}</path>
+       
+      ...
+   </configuration>
+</plugin>
+```
+
+Detailed explanation of the WAR deployment configurations are listed in the following table.
+
+Property | Description
+---|---
+`warFile` | Specify the war file location, optional if the war file location is: `${project.build.directory}/${project.build.finalName}.war`
+`path` | Specify context path, optional if you want to deploy to ROOT.
+
+## Supported Regions
```

Comment:
Shall we put "Supported Regions" and "Supported Pricing Tiers" under "Advanced Configurations"?